examples/first/README.md

```diff
@@ -0,0 +1,195 @@
+# Example#1
```

Comment:
The "for-human-eyes" doc is available here https://github.com/superstas/patron/tree/enhance-documentation-example1/examples/first